### PR TITLE
Fixes the Resize issue in activity pallete

### PR DESCRIPTION
**Pull request by Vishalk91-4**
Fixes issue #1303 

@llaske , if have added textarea element only in activity.css main , marked it as important and then forced through link onto the rest of the activities

https://github.com/llaske/sugarizer/assets/116670999/c8f7c257-682f-4a17-bfab-4976f72f8bd5

@llaske could you please review it, I think it's resolved

### CODE REVIEW

**Comment by llaske** [comment]
The lines fixed in `css/activity.css` are right. Nice.

BTW I'm not in favor of create a dependance between activities and Sugarizer Core so could you remove the `@import url(../../../css/activity.css)` and make the change in each activity.

**Comment by Vishalk91-4** [comment]
@llaske, Yes sure I'll do that

**Comment by Vishalk91-4** [comment]
@llaske, I've updated every activity.css with textarea, could you please review it, I hope this is what you wanted me to change

**Comment by Vishalk91-4** [comment]
@llaske , could you review this pull request please

File: activities/MediaViewer.activity/css/activity.css [hunk 1]
```diff
@@ -1,11 +1,16 @@
-#main-toolbar #activity-button {
```
Comment (llaske) [inline activities/MediaViewer.activity/css/activity.css:1]: This character should not be removed

**Comment by llaske** [comment]
> @llaske , could you review this pull request please

You've keep the generic fix in each CSS file:

```
textarea {
	resize: vertical !important;
	max-height: 70vh !important;
}

```
A I said, this fix could have side effect for activity using `textarea` in other context than the activity palette.
I thought the idea was to repeat the specific fix - change only style for the text area in the activity palette - in each file : 

```
.palette textarea#description.expand {
  resize: vertical !important;
  max-height: 70vh !important;
}

#description .row expand {
  resize: vertical !important;
  max-height: 70vh !important;; 
} 

textarea #description.expand {
  resize: vertical !important;
  max-height: 70vh !important;
}
```

Sorry if it was not clear in my comment.

**Comment by Vishalk91-4** [comment]
@llaske , sure I'll do that, actually it's my first time contributing so I think, I took it wrong

**Comment by llaske** [comment]
> @llaske , sure I'll do that, actually it's my first time contributing so I think, I took it wrong

Your first contribution is welcome!

**Comment by Vishalk91-4** [comment]
@llaske 
Do you want a fix like this 
image
in activitypallete.html file of each activity and this css change
image
for each activity file

Sorry if you meant anything else

**Comment by llaske** [comment]
> Do you want a fix like this
Nope.

To be more clear:
The issue is in the activity palette which is part of SugarWeb. So the fix should be done in SugarWeb (not in the activity)
The issue could be fixed by changing the style of the textarea used by the activity palette like that:

```
 .palette textarea#description.expand {
  resize: vertical !important;
  max-height: 70vh !important;
}

#description .row expand {
  resize: vertical !important;
  max-height: 70vh !important;; 
} 

textarea #description.expand {
  resize: vertical !important;
  max-height: 70vh !important;
}
```

This fix will apply only to the targeted object so there is no risk of side effects on others textarea in activity.
The fix is in the CSS is enough, no need to force styles in the HTML.

This fix should be done in the sugar-96dpi.css file of SugarWeb. Because SugarWeb component is duplicated in each activity, the fix should be applied to each activity directory.

**Comment by Vishalk91-4** [comment]
@llaske, could you check it now, whether it still cause dependencies on others or not

**Comment by llaske** [comment]
Sorry @Vishalk91-4, sound like I didn't view your PR.
Sounds correct now. Could you fix the conflict and extend the fix to new activities in dev branch?

**Comment by Vishalk91-4** [comment]
> Sorry @Vishalk91-4, sound like I didn't view your PR. Sounds correct now. Could you fix the conflict and extend the fix to new activities in dev branch?

Yes Sure, doing that right away

**Comment by Vishalk91-4** [comment]
Closing this PR, created a fresh PR #1522

**Comment by Vishalk91-4** [comment]
Closing this PR, created a fresh PR #1522

**Pull request closed without merging**